src/build-babel-preset-env.js

```diff
@@ -0,0 +1,218 @@
+import pluginList from "babel-preset-env/data/plugins.json";
```

Comment:
Just to separate babel-preset-env stuff with the scripts for building standalone version. But after last changes we have just 3 lines for the last, so I'll revert this.